src/handlers/GraphQLHandler.ts

```diff
@@ -211,4 +221,33 @@ Consider naming this operation or using "graphql.operation" request handler to i
     console.log('Response:', loggedResponse)
     console.groupEnd()
   }
+
+  /**
+   * Checks if the given operation name is matching with handler specifications
+   */
+  private hasMatchingOperationName(operationName: string | undefined) {
+    if (this.isGraphqQLOperationHandler()) {
+      return true
+    }
+
+    if (!operationName) {
+      return false
+    }
+
+    return this.info.operationName instanceof RegExp
+      ? this.info.operationName.test(operationName)
+      : operationName === this.info.operationName
+  }
+
+  /**
+   * Checks if the handler is a "graphql.operation" handler.
+   * I.e accepts all type of operations
+   */
+  private isGraphqQLOperationHandler() {
+    return (
+      this.info.operationType === 'all' &&
```

Comment:
I think it should be enough to rely on `this.info.operationType === 'all'` to reliably determine the `graphql.operation` handler, as it's its distinct value. We may drop the `isGraphQLOperationHandler` method altogether and adjust the error `if` statement to ensure we don't print the error when in the operation handler.